Contents/Sketch/styles.js

```diff
@@ -43,8 +43,7 @@ var applyStyles = function (newStyles, sharedStyles) {
     var existingStyles = sharedStyles.objects();
 
     while(existingStyles.count() > 0) {
-      style = existingStyles.objectAtIndex(0);
-      [existingStyles removeObject:style];
+      [sharedStyles removeSharedObjectAtIndex:0];
```

Comment:
This doesn't seem to work. If I run the plugin again I have duplicate styles.
![image](https://cloud.githubusercontent.com/assets/727778/17226742/b5c6c6b6-54bf-11e6-8633-2c75e6bc8954.png)